config/prometheus/monitor.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     app.kubernetes.io/name: aibrix
     app.kubernetes.io/managed-by: kustomize
   name: controller-manager-metrics-monitor
-  namespace: system
```

Comment:
copied review comment https://github.com/vllm-project/aibrix/pull/1112#discussion_r2155806524
> it will be overritten by kustomize if prometheus is under config/default.
> would this be applied separately or along with default?

Reply:
No, it's not a problem anymore since we've removed the duplicate ServiceMonitor.